README.md

```diff
@@ -106,10 +106,7 @@ Test certificates and instructions for generating certificates can be found in [
 npm run deploy
 ```
 
-This step is performed automatically when the `main` branch is updated:
-
-- https://staging-dot-mdn-bcd-collector.appspot.com/ is always deployed.
```

Comment:
I have removed the staging version. While https://staging-dot-mdn-bcd-collector.appspot.com/ still exists, it's the same as https://anything-dot-mdn-bcd-collector.appspot.com/ and just serves the same thing as https://mdn-bcd-collector.appspot.com/.